pkg/validation/policy/validate.go

```diff
@@ -1683,15 +1683,13 @@ func validateNamespaces(validationFailureActionOverrides []kyvernov1.ValidationF
 	return nil
 }
 
-func checkForScaleSubresource(ruleTypeJson []byte, allKinds []string, warnings *[]string) {
+func checkForScaleSubresource(ruleTypeJson []byte, allKinds []string) {
```

Comment:
Do we need this check if we want to remove this warning completely? Do we need to highligh this in best practice as [suggested](https://github.com/kyverno/kyverno/issues/9840#issuecomment-1988313290)?

Reply:
I’m not sure I fully understand what _Best Practices_ means in this context. Could you clarify?

Reply:
We could add a note to the subresource section in https://kyverno.io/docs/writing-policies/match-exclude/.